src/components/02_atoms/LoadComponent/LoadComponent.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { InputLabel, FormControl, FormGroup, FormControlLabel, Checkbox, FormHelperText } from '@material-ui/core';
+
+class LoadComponent extends React.Component {
+  state = {
+    component: null,
+  };
+
+  static propTypes = {
+    name: PropTypes.string.isRequired,
+    component: PropTypes.string.isRequired,
+    render: PropTypes.func.isRequired,
+  };
+
+  componentDidMount() {
+    if (typeof this.props.component === 'string') {
+      // Cheat and load "React"
+      window.define('react', () => React);
```

Comment:
This is ugly, but it works!

Reply:
Probably worth abstracting this out into its own thing so we only have to do it once for the first component that's 'loaded'.